src/Tribe/PUE/Checker.php

```diff
@@ -741,13 +758,13 @@ public function get_domain() {
 		/********************** General Functions **********************/
 
 		/**
-		 * Compile  a list of addons
+		 * Compile a list of addons.
 		 *
 		 * @param array $addons List of addons.
 		 *
-		 * @return array list of addons
+		 * @return array List of compiled addons.
```

Comment:
```suggestion
		 * @return array List of addons.
```
The way you have it it says "here is a list, each of these addons is compiled"

There's no addon "compiling" going on here - just creating a list.